### PR TITLE
feat: Improved country code in contact form view.

### PR DESCRIPTION
**Pull request by iamsivin**
# Pull Request Template

## Description

Fixes https://linear.app/chatwoot/issue/CW-350/improved-country-code-in-contact

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
https://www.loom.com/share/8581376cd395490894cc333c8e377e51

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by muhsin-k** [comment]
@iamsivin 
1. The modal is not closing when clicking on outside.
2. Add a cursor pointer for the button.
3. Can we add keyboard actions to the select?

https://user-images.githubusercontent.com/12408980/230879699-fad3428a-1f4b-4f43-905c-53857912bd5a.mov

File: app/javascript/dashboard/components/widgets/forms/PhoneInput.vue [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+          `+${number.countryCallingCode}`,
+          ''
+        );
+      }
+    },
+  },
+  mounted() {
+    setTimeout(() => {
+      this.setActiveCountry();
+    }, 10);
```
Comment (muhsin-k) [inline app/javascript/dashboard/components/widgets/forms/PhoneInput.vue:142]: What is the need for `setTimeout` here?

**Comment by iamsivin** [comment]
> @iamsivin
> 
> 1. The modal is not closing when clicking on outside.
> 2. Add a cursor pointer for the button.
> 3. Can we add keyboard actions to the select?

1. The v-clickaway is not working on top of the modal. The issue is that from the z-index, we need to remove the z-index value of the contact form. If we remove it, the UI will break. Do you have any idea about this issue? 
CC: @scmmishra @nithindavid 

2. Done
3. Done

File: app/javascript/dashboard/components/widgets/forms/PhoneInput.vue (hunk 1 above)
Comment (iamsivin) [reply app/javascript/dashboard/components/widgets/forms/PhoneInput.vue]: @muhsin-k Added it at the time of doing forgot to remove it. Removed it now. https://github.com/chatwoot/chatwoot/pull/6801/commits/caff3147418e1a9629bee966d4f96c1df9dbbd9a

**Review by muhsin-k — APPROVED** [review]

**Comment by nithindavid** [comment]
Tested and working as expected.

**Pull request merged**